rpy2/rinterface.py

```diff
@@ -879,6 +880,25 @@ def initr_checkenv() -> typing.Optional[int]:
 initr = initr_checkenv
 
 
+def _update_R_ENC_PY():
+    conversion._R_ENC_PY[openrlib.rlib.CE_UTF8] = 'utf-8'
+
+    l10n_info = tuple(baseenv['l10n_info']())
+    if platform.system() == 'Windows':
+        val_latin1 = 'cp{}'.format(l10n_info[3])
```

Comment:
Is this correct when the codepage is not 1252 e.g. 950? From https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/l10n_info I get the impression that l10n_info can return other codepages than 1252. But I read the man page documentation mentioned in #754 as if for R ≥ 3.5.0 it is always interpreted as cp1252 (on Windows?).

Reply:
I don't know. My interpretation of the documentation was that 1252 was given as an example, but that's not necessarily the only one.

`l10n_info` returns more information on Windows than on Linux so I solely rely on reading the doc (or R's source code). The doc page you are pointing indicates that the `codepage` returned is the one R is using (and is not necessarily the one generally used by the Windows OS R is running on), doesn't it?

Reply:
The CI is now working again with Windows (ABI mode) and I added an automated for what this PR is fixing. In the interest of time, I'll merge it. I'd like to release 3.4.0 within the next days.